src/components/Menu/VersionSwitcher/index.tsx

```diff
@@ -2,30 +2,6 @@ import Link from "next/link";
 import {ActiveSwitchStyle, SwitchStyle} from "./styles";
 import directory from "../../../directory/directory";
 
-const ui = directory["ui"].items;
-const uiLegacy = directory["ui-legacy"].items;
-const uiLegacyPaths = [];
-const uiPaths = [];
```

Comment:
Is this safe to remove? Used below on line 58

Reply:
let me check that

Reply:
@timngyn  - can you help me with checking what do I need to remove from this file specifically? This is looking more surgical to me.

Reply:
Just made a new PR here with the changes I think we should remove: https://github.com/aws-amplify/docs/pull/4855/files

Since we're removing the UI Library pages completely we can just remove the `VersionSwitcher` component because that was hardcoded to only the UI Library pages and was not being used anywhere else

Reply:
merged now - thank you, Tim!